docs/test-pr-guide.md

```diff
@@ -0,0 +1,97 @@
+
+5. Run the following commands to get to the PR branch, where `<contributor>` is the GitHub username of the contributor that submitted the PR:
+
+```
+git checkout -b <contributor>-<branch-name> develop
+git pull https://github.com/<contributor>/mentorship-backend/ <branch-name>
+```
+
+6. Verify the code addition/deletions in the PR.
+
+7. Reproduce the issue and test the fix.
+
+8. Get screenshots/gifs of before and after the fix and attach them to the PR comment.
+
+<h1>Steps to test multiple features (User Registration, List users, ...) of the Mentorship System.</h1>
+
+<h2>Registering users</h2>
+
+Register 3 users. Follow instructions in 
+[this video](https://www.youtube.com/watch?v=xRZrdR47R-w&feature=youtu.be&t=672)
+
+The users used here are 
+
+* testusera
+* testuserb
+* testuserc
+
+![Register user screenshot](https://user-images.githubusercontent.com/26095715/79673508-4a582180-81a8-11ea-87ce-fe0bfda82fea.png)
+
+<h2>Login as the users and capture the access tokens</h2>
+
+Steps to login and capture the token are shown in 
+[this video](https://www.youtube.com/watch?v=xRZrdR47R-w&feature=youtu.be&t=672)
+
+![Login user screenshot](https://user-images.githubusercontent.com/26095715/79673507-4a582180-81a8-11ea-96ff-2268733e9673.png)
+
+<h2>Get the user ID's of the users</h2>
+
+In this example they are:
+
+* testusera - ID: 120
+* testuserb - ID: 121
+* testuserc - ID: 122
+
+![User ID screenshot1](https://user-images.githubusercontent.com/26095715/79673506-49bf8b00-81a8-11ea-8bf3-0dffd8b269f5.png)
+
+![User ID screenshot2](https://user-images.githubusercontent.com/26095715/79673505-49bf8b00-81a8-11ea-89f2-687913a238fe.png)
+
+<h2>Create a mentorship relation between 2 users</h2>
+<h3>Send a mentorship request</h3>
+
+In this example testusera is the mentee and testuserb is the mentor and testusera is sending the request. The steps to do this is explained in this video.
+
+[Demo Video](https://www.youtube.com/watch?v=xRZrdR47R-w&feature=youtu.be&t=672)
+
+![Mentorship Request screenshot1](https://user-images.githubusercontent.com/26095715/79673504-49bf8b00-81a8-11ea-9a1e-c136cb345440.png)
+
+<h3>Check pending requests</h3>
+
+Check pending requests for testusera. The relation should be in state 1 (pending). Note the request ID. Here it is 13.
+
+![Pending request testusera screenshot1](https://user-images.githubusercontent.com/26095715/79673502-4926f480-81a8-11ea-8695-882117830657.png)
+
+![Pending request testusera screenshot2](https://user-images.githubusercontent.com/26095715/79673501-4926f480-81a8-11ea-9d78-25704487ce28.png)
+
+<h3>Accept the mentorship request</h3>
+
+testuserb accepts the mentorship request from testusera.
+
+![Accepted request screenshot](https://user-images.githubusercontent.com/26095715/79673500-4926f480-81a8-11ea-9c97-bac8cc800f85.png)
+
+<h3>List current relations of a user</h3>
+
+List the current relationships of testusera. The relation should be state 2 (accepted).
```

Comment:
```suggestion
List the current relationships of _testusera_. The relation should be state 2 (accepted).
```
I forgot to mention this, having these names of user a little different to be easily skimmed through, what do you think? @rpattath

Reply:
Can you do this for everytime you mention in a sentence the username?